portality/settings.py

```diff
@@ -706,7 +719,7 @@
         "article" : {
             "auth" : False,
             "role" : None,
-            "query_validator" : "public_query_validator",
+            "query_validators" : ["non_public_fields_validator", "public_query_validator"],
```

Comment:
I'm happy to have two separate validators here chained together, OR for us to update the `public_query_validator` so that it can do both jobs.  If we're having 2 separate validators, maybe we should rename the `public_query_validator` to something more descriptive of what it does (which is to check paging parameters).

Reply:
`non_public_fields_validator` is used at the places where `public_query_validator` not used. For example for Publisher application query (Line no 755) So I made them separate validators.